docker-compose.ci.yml

```diff
@@ -0,0 +1,9 @@
+tests:
+  build: .
```

Comment:
Ah, that's exactly what you told me this morning but I missed the significance. Now I know what you mean.